cmd/podman/containers/container.go

```diff
@@ -20,7 +20,7 @@ var (
 		RunE:             validate.SubCommandExists,
 	}
 
-	containerConfig = util.DefaultContainerConfig()
```

Comment:
please use `registry.PodmanConfig().ContainersConfDefaultsRO` in all cmd/podman/... files instead